index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <section class="topbar">
+    <div class="logo">Abdul Nganje</div>
+    <div class="menu-icon"><img src="images/Union.svg" alt=""></div>
+    </section>
```

Comment:
- The contents of the toolbar section should put inside a header tag.
- Your logo should have an a tag
- Your hamburger menu should be inside a nav tab
- Please follow the figma design for font, color and margin/padding